Assets/Scripts/Gameplay/CharacterController2D.cs

```diff
@@ -54,6 +56,6 @@ private void Movement(Vector3 pMovementVelocity)
     //OnInteract is called when the Input System gets an input for Interact (check input actions)
```

Comment:
Can you use /// Summary for this comment instead of using a regular // comment? Might it also not be better to use an event for this?